src/CommandingOfficers/AWBW/BH/SturmGrimmBW.java

```diff
@@ -0,0 +1,152 @@
+package CommandingOfficers.AWBW.BH;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import CommandingOfficers.*;
+import CommandingOfficers.AWBW.AWBWCommander;
+import CommandingOfficers.AWBW.AWBWValueFinders;
+import Engine.GameScenario;
+import Engine.XYCoord;
+import Engine.Combat.DamagePopup;
+import Engine.Combat.StrikeParams;
+import Engine.Combat.StrikeParams.BattleParams;
+import Engine.GameEvents.GameEvent;
+import Engine.GameEvents.GameEventQueue;
+import UI.UIUtils;
+import Units.UnitContext;
+import Terrain.GameMap;
+import Terrain.MapMaster;
+
+public class SturmGrimmBW extends AWBWCommander
+{
+  private static final long serialVersionUID = 1L;
+
+  private static final CommanderInfo coInfo = new instantiator();
+  public static CommanderInfo getInfo()
+  {
+    return coInfo;
+  }
+  private static class instantiator extends CommanderInfo
+  {
+    private static final long serialVersionUID = 1L;
+    public instantiator()
+    {
+      super("Sturm", UIUtils.SourceGames.AWBW, UIUtils.BH, "GBW");
+      infoPages.add(new InfoPage(
+            "Sturm (Grimm BW)\n"
+          + "AWBW Sturm, but with Grimm's stats (matching AW1's D2D).\n"
+          + "Movement cost over all terrain is reduced to 1, except in cold weather.\n"
+          + "Units gain +30/-20 stats.\n"));
```

Comment:
In the other AWBW Sturm, the unit stat line is not separated from the cold-weather line, like it is here. Recommend adding a newline.